README.md

```diff
@@ -65,261 +77,140 @@ implementation "com.google.android.gms:play-services-ads-identifier:18.0.1"
 ---
 In your application's `onCreate` method, start the Chartboost Mediation SDK by providing your app's app id, its app signature, and passing a _HeliumSdkListener_
 
-```java Java
-HeliumSdk.start(
-    MyActivityContext.this,
-    myHeliumAppId,
-    myHeliumAppSignature,
-    new HeliumInitializationOptions(),
-    new HeliumSdk.HeliumSdkListener() {
-        @Override
-        public void didInitialize(Error error) {
-            if (error != null) {
-                Log.d(TAG,"Chartboost Mediation SDK failed to initialize. Reason: " + error.getMessage());
-            } else {
-                //SDK Started,
-                Log.d(TAG,"Chartboost Mediation SDK initialized successfully");
+```kotlin Kotlin
```

Comment:
If we are updating the code in the README.md, we may need to update the text as well as it is referencing old names.

Reply:
https://github.com/ChartBoost/chartboost-mediation-android-sdk-demo/pull/39/commits/6c2d00d5572474dfef9c895c48988c429f192777